### PR TITLE
fix(FInput): 修复复合型输入框样式问题

### PR DESCRIPTION
**Pull request by zym19960704**
**What kind of change does this PR introduce?** (check at least one)

-   [x] Bugfix
-   [ ] Feature
-   [ ] Code style update
-   [ ] Refactor
-   [x] Docs
-   [ ] Build-related changes
-   [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

-   [ ] Yes
-   [x] No

If yes, please describe the impact and migration path for existing applications:

**Related issue (if exists):**

**Other information:**

### CODE REVIEW

File: components/input/style/mixin.less [hunk 1]
```diff
@@ -19,25 +19,65 @@
 }
 
 .group-append() {
+    .hidden-left-border();
```
Comment (ocean-gao) [inline components/input/style/mixin.less:22]: 之前命名不规范，可以left统一为prepend, right统一为append
hidden-prepend-border();

File: components/input/style/mixin.less [hunk 2]
```diff
@@ -19,25 +19,65 @@
...
+    div.@{cls-prefix}-select-cascader .@{cls-prefix}-select-trigger,
     div.@{cls-prefix}-select:hover .@{cls-prefix}-select-trigger,
     div.@{cls-prefix}-cascader:hover .@{cls-prefix}-select-trigger {
         .border-left-pure();
     }
 }
+
+
 .group-prepend() {
+    .hidden-right-border();
```
Comment (ocean-gao) [inline components/input/style/mixin.less:35]: 命名规范

File: components/input/style/mixin.less [hunk 3]
```diff
@@ -19,25 +19,65 @@
...
     div.@{cls-prefix}-select .@{cls-prefix}-select-trigger,
     div.@{cls-prefix}-cascader .@{cls-prefix}-select-trigger,
+    div.@{cls-prefix}-select-cascader .@{cls-prefix}-select-trigger,
     div.@{cls-prefix}-select:hover .@{cls-prefix}-select-trigger,
     div.@{cls-prefix}-cascader:hover .@{cls-prefix}-select-trigger  {
         .border-right-pure();
     }
 }
+
+.light-border {
```
Comment (ocean-gao) [inline components/input/style/mixin.less:46]: 这里是函数式用法，所以建议加上函数标识，虽然运行是不影响的 .light-border()

File: components/input/style/mixin.less [hunk 4]
```diff
@@ -19,25 +19,65 @@
...
+    div.@{cls-prefix}-select .@{cls-prefix}-select-trigger,
+    div.@{cls-prefix}-cascader .@{cls-prefix}-select-trigger,
+    div.@{cls-prefix}-select-cascader .@{cls-prefix}-select-trigger {
+        border: 1px solid var(--f-primary-color);
+    }
+}
+
+.hidden-right-border {
```
Comment (ocean-gao) [inline components/input/style/mixin.less:54]: 同上

File: components/input/style/mixin.less (hunk 3 above)
Comment (ocean-gao) [reply components/input/style/mixin.less:46]: 这里命名为 hover-light-border()

File: components/input/style/mixin.less [hunk 5]
```diff
@@ -19,25 +19,65 @@
...
+    div.@{cls-prefix}-select .@{cls-prefix}-select-trigger,
+    div.@{cls-prefix}-cascader .@{cls-prefix}-select-trigger,
+    div.@{cls-prefix}-select-cascader .@{cls-prefix}-select-trigger {
+        border-right: 1px solid transparent;
+    }
+}
+
+.hidden-left-border {
```
Comment (ocean-gao) [inline components/input/style/mixin.less:62]: 命名规范同上

File: components/input/style/mixin.less [hunk 6]
```diff
@@ -19,25 +19,65 @@
...
+    div.@{cls-prefix}-select .@{cls-prefix}-select-trigger,
+    div.@{cls-prefix}-cascader .@{cls-prefix}-select-trigger,
+    div.@{cls-prefix}-select-cascader .@{cls-prefix}-select-trigger {
+        border-left: 1px solid transparent;
+    }
+}
+
+.select-focus-border-light() {
```
Comment (ocean-gao) [inline components/input/style/mixin.less:70]: 这里命名为 focus-light-border()

**Comment by zym19960704** [comment]
您好，您的邮件我已收到。我将尽快给您回复，谢谢您的来件。

**Pull request merged**